docs/source/conf.py

```diff
@@ -81,15 +81,11 @@
 
 # The theme to use for HTML and HTML Help pages.
 html_theme = "furo"
-
+# hide rubicon because it's already in the logo, but will still get pulled into the tab
+html_title = f"<code class='hidden'>rubicon</code> <div class='version'> v{version}</div>"
```

Comment:
🔥 🔥 🔥 

you can probably just make the hidden class a regular div rather than the code one

Reply:
yea, good call with this hidden tag -- ended up working out perfectly